### ISSUE TITLE
Error fetching GraphQL query 'EmbeddingUMAPQuery' with variables '{"id":"RW1iZWRkaW5nRGltZW5zaW9uOjA=","timeRange":{"start":"2022-08-03T20:30:00.000Z","end":"2022-08-05T20:30:00.000Z"},"minDist":0,"nNeighbors":30,"nSamples":500,"minClusterSize":10,"clusterMinSamples":1,"clusterSelectionEpsilon":0,"fetchDataQualityMetric":false,"fetchPerformanceMetric":false,"dataQualityMetricColumnName":null,"performanceMetric":"accuracyScore"}': [{"message":"name '_HAS_DEFAULT_FACTORY' is not defined","locations":[{"line":18,"column":7}],"path":["embedding","UMAPPoints"]}][BUG]

### ISSUE THREAD

**Issue by mallikarjunyadav27** [Arize-ai/phoenix#1023]

**Comment by mallikarjunyadav27** [comment]
Error fetching GraphQL query 'EmbeddingUMAPQuery' with variables '{"id":"RW1iZWRkaW5nRGltZW5zaW9uOjA=","timeRange":{"start":"2022-08-03T20:30:00.000Z","end":"2022-08-05T20:30:00.000Z"},"minDist":0,"nNeighbors":30,"nSamples":500,"minClusterSize":10,"clusterMinSamples":1,"clusterSelectionEpsilon":0,"fetchDataQualityMetric":false,"fetchPerformanceMetric":false,"dataQualityMetricColumnName":null,"performanceMetric":"accuracyScore"}': [{"message":"name '_HAS_DEFAULT_FACTORY' is not defined","locations":[{"line":18,"column":7}],"path":["embedding","UMAPPoints"]}]

**Comment by mikeldking** [comment]
closing in favor of #1041

**Comment by mikeldking** [comment]
@mallikarjunyadav27 Any chance you could provide us with:

- OS
- output of pip freeze
- your python version

Thanks! We will try to unblock you in the next week or so.
